spec/requests/api/v1/docs/pages_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "api/v1/pages" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:page) { create(:page) }
+  let(:user) { api_secret.user }
+
+  before do
+    user.add_role(:admin)
+
+    allow(FeatureFlag).to receive(:enabled?).with(:api_v1).and_return(true)
```

Comment:
Do we need this line? I've not seen it in the request specs but not on the docs specs.

Reply:
It's been unnecessary since [August](https://github.com/forem/forem/pull/18213), but it seems I've been blindly following a pattern still used in other API specs. I'll clean up! 🧹